guides-and-standards/guide-choosing-software-dependencies.md

```diff
@@ -0,0 +1,59 @@
+  - Easy: installable from another package repository with another package manager like pip (python) or `install_packages()` (R)
+  - Somewhat easy: Installable from GitHub with a package manager
+  - Somewhat easy: Has a dockerfile and an available Docker container
+  - Somewhat difficult: has an environment yaml file or requirements.txt file that lists versioned dependencies and has installation instructions that involve cloning the repository
+  - Difficult: git clone & compile from source
+  - Difficult: No installation instructions
+
+### Nice — but not necessary if you’re willing to do the leg work to make sure the tool works
+
+- **The tool has tests.** Tests ensure the accuracy and quality of a software tool. See [this AUG](https://training.arcadiascience.com/arcadia-users-group/20231104-testing-concepts/lesson/) for more about software tests.
+  - Hint: tests will often be a `test/` or `tests` folder, either in the root of the repository or
+- **The software is maintained.**
+  - Hint: there is a difference in the timestamp between the first and last commit and the last commit occurred recently (~months)
+  - Hint: there are responses to issues
+- **The software is well-written.**
+  - Hint: no `.DS_Store` files visible in any folder of the repository
+  - Hint: the repository has a reasonable structure
+  - Hint: the software is written in functions
+  - Hint: there are no absolute paths in the software
+  - Hint: the software is packaged with or has documentation for obtaining demo data so you can quickly try to run the tool.
+- **Other people use the software.** This signals that other people believe in the method and that other people were able to install and use the software.
+  - Hint: the GitHub repository has a lot of stars
+  - Hint: the paper publishing the software has been cited frequently by other projects that used the tool.
+
+## Ideas for finding quality software
+
+- Search [conda](https://anaconda.org/anaconda/repo), [PyPI](https://pypi.org/), or other software package repositories with keywords relevant to your task. The search engines for different software package repositories are different so your mileage will vary. However, if you can find a tool this way, the chances are good that it will be easy to install and mediocre that the software will be high quality.
+- Search Google Scholar with your keywords plus "GitHub." This may highlight studies that have published software. You can filter to recent papers or select papers that are highly cited.
+- Look at the [nf-core modules](https://github.com/nf-core/modules) repository. If a tool makes it into that repo, it can be dockerized, installed on Linux, run on the command line, and is usually fairly high quality. Make sure to check the license to make sure you can use the tool. While you're there, you can look to see if an existing [nf-core workflow](https://github.com/nf-core) already meets your full analysis needs.
```

Comment:
Since docker is mentioned here, maybe mention biocontainers and/or quay.io? There's a small amount of people I would imagine that would need to know that, but might change over time

Reply:
i'll switch to "containerized" so it isn't specific